packages/headless/src/controllers/result-list/headless-result-list.ts

```diff
@@ -96,10 +98,9 @@ export function buildResultList(
       const state = engine.state;
 
       return {
+        ...searchStatus.state,
```

Comment:
Same comment as above on just including the missing property instead of the entire state object, to prevent additions to the searchStatus controller from increasing the number of times other controllers rerender.

Reply:
Here the results are already part of the controller's state, it won't trigger more re-renders. And, like the query summary, the additional state properties are useful, they will be used.